test/Worker.Extensions.DurableTask.Tests/FunctionsDurableTaskClientTests.cs

```diff
@@ -53,5 +55,93 @@ public async void TerminateDoesNotThrow()
+        /// </summary>
+        [Fact]
+        public void CreateHttpManagementPayload_WithHttpRequestData()
+        {
+            string requestUrl = "http://localhost:7075/orchestrators/E1_HelloSequence";
+            FunctionsDurableTaskClient client = this.GetTestFunctionsDurableTaskClient();
+            string instanceId = "testInstanceIdWithRequest";
+            var context = new TestFunctionContext();
+            var request = new TestHttpRequestData(context, new Uri(requestUrl));
+
+            dynamic payload = client.CreateHttpManagementPayload(instanceId, request);
+
+            AssertHttpManagementPayload(payload, "http://localhost:7075/runtime/webhooks/durabletask", instanceId);
+        }
+
+        private static void AssertHttpManagementPayload(dynamic payload, string BaseUrl, string instanceId)
+        {
+            Assert.Equal(instanceId, payload.id);
+            Assert.Equal($"{BaseUrl}/instances/{instanceId}", payload.purgeHistoryDeleteUri);
+            Assert.Equal($"{BaseUrl}/instances/{instanceId}/raiseEvent/{{eventName}}", payload.sendEventPostUri);
+            Assert.Equal($"{BaseUrl}/instances/{instanceId}", payload.statusQueryGetUri);
+            Assert.Equal($"{BaseUrl}/instances/{instanceId}/terminate?reason={{{{text}}}}", payload.terminatePostUri);
+            Assert.Equal($"{BaseUrl}/instances/{instanceId}/suspend?reason={{{{text}}}}", payload.suspendPostUri);
+            Assert.Equal($"{BaseUrl}/instances/{instanceId}/resume?reason={{{{text}}}}", payload.resumePostUri);
+        }
+    }
+
+    /// <summary>
+    /// A minimal implementation of FunctionContext for testing purposes.
+    /// It's used to create a TestHttpRequestData instance, which requires a FunctionContext.
+    /// </summary>
+    public class TestFunctionContext : FunctionContext
+    {
+        public override string InvocationId => throw new NotImplementedException();
+        public override string FunctionId => throw new NotImplementedException();
+        public override TraceContext TraceContext => throw new NotImplementedException();
+        public override BindingContext BindingContext => throw new NotImplementedException();
+        public override RetryContext RetryContext => throw new NotImplementedException();
+        public override IServiceProvider InstanceServices { get; set; } = new EmptyServiceProvider();
+        public override FunctionDefinition FunctionDefinition => throw new NotImplementedException();
+        public override IDictionary<object, object> Items { get; set; } = new Dictionary<object, object>();
+        public override IInvocationFeatures Features => throw new NotImplementedException();
+    }
+
+    /// <summary>
+    /// A minimal implementation of IServiceProvider for testing purposes.
+    /// </summary>
+    public class EmptyServiceProvider : IServiceProvider
+    {
+        public object GetService(Type serviceType) => null;
```

Comment:
To make the warning go away, change `null` to `null!`.
```suggestion
        public object GetService(Type serviceType) => null!;
```